bin/meshtasticd.service

```diff
@@ -1,12 +1,16 @@
 [Unit]
 Description=Meshtastic Native Daemon
 After=network-online.target
+StartLimitInterval=200
+StartLimitBurst=5
 
 [Service]
 User=root
 Group=root
 Type=simple
 ExecStart=/usr/sbin/meshtasticd
+Restart=always
```

Comment:
Perhaps on-failure makes more sense than always?

Reply:
That would need an audit of whether we properly use return codes for all of our exit conditions.